sherpa/models/basic.py

```diff
@@ -40,14 +40,36 @@
 __all__ = ('Box1D', 'Const1D', 'Cos', 'Delta1D', 'Erf', 'Erfc', 'Exp', 'Exp10',
            'Gauss1D', 'Log', 'Log10', 'LogParabola', 'NormGauss1D', 'Poisson',
            'Polynom1D', 'PowLaw1D', 'Scale1D', 'Sin', 'Sqrt', 'StepHi1D',
-           'StepLo1D', 'Tan', 'Box2D', 'Const2D', 'Delta2D', 'Gauss2D',
-           'SigmaGauss2D',
+           'StepLo1D', 'Tan', 'Voigt1D',
+           'Box2D', 'Const2D', 'Delta2D', 'Gauss2D', 'SigmaGauss2D',
            'NormGauss2D', 'Polynom2D', 'Scale2D', 'UserModel', 'TableModel',
            'Integrate1D')
 
 DBL_EPSILON = numpy.finfo(numpy.float).eps
 
 
+class Voigt1D(RegriddableModel1D):
+    """Voigt1D Profile
+    http://publikationen.badw.de/de/003395768
+    """
+
+    def __init__(self, name='voigt1d'):
+        # fwhm_g = 2 * alpha
+        # fwhm_l - 2 * gamma        
+        self.fwhm_g = Parameter(name, 'fwhm_g', 0.05, min=0.0)
```

Comment:
We should use to match the Gauss1D and Lorentz1D models [although technically we should reduce the default values of fwhm_l/g from 10 to something smaller)

        self.fwhm_g = Parameter(name, 'fwhm_g', 10, tinyval, hard_min=tinyval)
        self.fwhm_l = Parameter(name, 'fwhm_l', 10, 0, hard_min=0)

[I you set fwhm_g to 0 you get nan's out, so we should stop that]